### PR TITLE
Update package json

### LINKED ISSUE #35: Update `package.json`
- version number will be `v0.10.0`;
https://github.com/OpenSTFoundation/mosaic.js/blob/2f5fbb79e5dbb6c6276b299e1e92e535516e95dd/package.json#L3

- update description

### PR DESCRIPTION
**Pull request by 0xsarvesh**
Fixes #35 

1. Updated version and description.
2. Removed unused dependency.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@openstfoundation/mosaic.js",
-  "version": "0.1.0",
-  "description": "NPM package that provides mosaic contract abis and bins. This package also helps in mosaic chain setup.",
+  "version": "0.10.0",
+  "description": "NPM package that provides mosaic contract abis and bins. This package also helps in mosaic chain setup and interaction with mosaic contracts.",
```
Comment (schemar) [inline package.json]: This package is not supposed to help with chain setup. Only with Gateway setup. If I remember correctly.

@benjaminbollen ?

File: package.json [hunk 2]
```diff
@@ -32,9 +32,8 @@
   },
   "dependencies": {
     "@openstfoundation/mosaic-contracts": "^0.10.0-alpha.1",
-    "bignumber.js": "4.1.0",
+    "bn.js": "4.11.8",
     "js-scrypt": "^0.2.0",
-    "node-cmd": "3.0.0",
     "shelljs": "0.8.1",
```
Comment (schemar) [inline package.json]: For what do we use `shelljs`?

File: package.json [hunk 3]
```diff
@@ -45,18 +44,14 @@
     "@babel/preset-env": "7.1.0",
     "@babel/runtime": "7.0.0",
     "babel-loader": "8.0.2",
```
Comment (schemar) [inline package.json:45]: Do we keep babel because we plan to use it in the future?

**Review by schemar — APPROVED** [review]
LGTM :shipit:

File: package.json (hunk 3 above)
Comment (hobofan) [reply package.json:45]: It is used in `webpack.config.js`

**Review by hobofan — CHANGES_REQUESTED** [review]
We should also add this to the package.json, so we only publish the required files:
```json
  "files": [
    "lib"
],
```

File: package.json (hunk 1 above)
Comment (0xsarvesh) [reply package.json]: I can see, it's been exposed in [index.js](https://github.com/OpenSTFoundation/mosaic.js/blob/ec5b334913d610f0fb37eb7fe91fc556da8c03b1/index.js#L37).

File: package.json (hunk 2 above)
Comment (0xsarvesh) [reply package.json]: I have removed it, I think it was used for the previous web pack thing.

**Review by hobofan — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (hobofan) [reply package.json]: Do we have a explanation somewhere that makes a clear distinction between "chain setup" and "gateway setup"? That the class is called `ChainSetup` in the package here probably doesn't help...

Comment (hobofan) [reply package.json]: Maybe go with the a bit vague `helps in mosaic setup` for now? 😬

Comment (0xsarvesh) [reply package.json]: `mosaic-setup` is fine. As it setup gateway and anchor both right?

Comment (schemar) [reply package.json]: Was discussed with @benjaminbollen.
This repo should not have a "do whole chain setup" thing. Instead, it should supply individual actions like "deploy gateway" and possibly even "deploy anchor". Side-chain setup with staked validators however will be done from the mosaic-contracts layer. mosaic.js supplies functionality to token owners (e.g. BenCoin BCO) who want to deploy gateways for their token onto an existing mosaic chain.

Comment (schemar) [reply package.json]: Long story short: I would not put "mosaic setup" in the description. The description would be that this package helps you to interact with mosaic related contracts. E.g. EIP20Gateway, etc.

Comment (schemar) [reply package.json]: And with interaction and deployment.

Comment (schemar) [reply package.json]: Which is not setup (think new side chain with validators setup from mosaic-contracts).

**Pull request merged**